Specs/DateTime/Chinese/DatePeriodParser.json

```diff
@@ -467,6 +467,82 @@
   ]
 }
 ,
+{
+  "Input": "时间从2016到2018",
+  "Context": {
+    "ReferenceDateTime": "2017-03-22T00:00:00"
+  },
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "从2016到2018",
+      "Type": "daterange",
+      "Value": {
+        "Timex": "(2016,2018,P2Y)",
+        "FutureResolution": {
+          "startDate": "2016-01-01",
+          "endDate": "2018-01-01"
```

Comment:
Hmmm. I see the issue.

Reply:
Can you add a code comment in the Parser method description explaining our default definition of interval? I.e. [start,end)

Reply:
For resolution, I'll update this comment later if we should use the format you have here, or another.

Reply:
To standardize things, the assumption is that all intervals are [start,end) and this matches the example here. 
Interval value in NumberRange, timex, and resolutions should always be non-inclusive of the end.
There is only one exception: for time, if the end of the range is midnight it will be represented as 23:59:59. Which doesn't apply here as this is Date only, not DateTime.